lib/src/main/java/org/team100/lib/config/Camera.java

```diff
@@ -32,7 +32,7 @@ public enum Camera {
     C("10000000a7c673d9",
             new Transform3d(
                     new Translation3d(0, 0, 0.75),
-                    new Rotation3d(0, Math.toRadians(30), 0))),
+                    new Rotation3d(0, Math.toRadians(30), Math.PI))),
```

Comment:
the translation of this camera seems wrong.  also could you make the enum names something that describes what the camera does?  there's no reason to use names like "A" "B" "C".